packages/cli/src/workflows/list/retrieveWorkflowDefinition/RetrieveWorkflowDefinition.definition.ts

```diff
@@ -46,11 +46,6 @@ export const RetrieveWorkflowDefinition: ICommandDefinition = {
             description: "To list the contents of a workflow definition stored in the UNIX file \"/user/dir/workflow.xml\"" +
             " including its steps and variables",
             options: "\"/user/dir/workflow.xml\" --list-steps --list-variables"
-        },
-        {
-            description: "To list the contents of a workflow definition stored in the z/OS data set \"USER.DATA.SET.XML\"" +
-            " including its steps and variables",
-            options: "--workflow-name \"testWorkflow\" --list-steps --list-variables"
         }
```

Comment:
Do we want the 2nd example deleted, or the `options` list corrected to match what the description says? I'm fine either way 😛